sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -1381,9 +1461,20 @@ public void finishBundle(FinishBundleContext c) throws Exception {
             // to the error queue. Bigtable will successfully write other failures in the batch,
             // so this exception should be ignored
             if (!(e.getCause() instanceof BatchingException)) {
+              throttlingMsecs.inc(new Duration(closeStart, Instant.now()).getMillis());
               throw e;
```

Comment:
Is this error due to resource exhaustion?

Reply:
it's server side failing in general that not due to bad data / bad client request, etc. Could be resource exhaust or server downtime, and all treated as throttled time here (as worker CPU time wasted)